types/env.go

```diff
@@ -20,14 +21,23 @@ type BlockInfo struct {
 	ChainID string `json:"chain_id"`
 }
 
+type ContractInfo struct {
+	// binary encoding of sdk.AccAddress of the contract, to be used when sending messages
```

Comment:
q: should this be "bech32 encoded address" instead?

Reply:
oh, you are right. I will update all the comments